LoRaEngine/LoraKeysManagerFacade/CreateEdgeDevice.cs

```diff
@@ -1,28 +1,28 @@
-//
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-//
-using System.IO;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.Azure.WebJobs;
-using Microsoft.Azure.WebJobs.Extensions.Http;
-using Microsoft.AspNetCore.Http;
-using Microsoft.Azure.WebJobs.Host;
-using Newtonsoft.Json;
-using System;
-using Microsoft.Azure.Devices;
-using Microsoft.Azure.Devices.Shared;
-using System.Collections.Generic;
-using System.Net;
-using System.Net.Http;
-using Microsoft.Extensions.Configuration;
-using System.Text;
 
 namespace CreateDeviceFunction
 {
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Net;
+    using System.Net.Http;
+    using System.Text;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.Devices;
+    using Microsoft.Azure.Devices.Shared;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Host;
+    using Microsoft.Extensions.Configuration;
+    using Newtonsoft.Json;
+
     public static class CreateEdgeDevice
     {
         [FunctionName("CreateEdgeDevice")]
+        [Obsolete]

```

Comment:
Why is this Method refered as Obsolete? I need it as part of my ARM template. Did you find another way of creating the devices?

Reply:
StyleCop requested me to add this [Obsolete] tag. This was treated as an Error by StyleCop...

Reply:
Removed the Obsolete tag.